fairscale/utils/testing.py

```diff
@@ -571,7 +571,7 @@ def forward(self, *input: Tensor, **kwargs: Any) -> Tensor:
 def get_cycles_per_ms() -> float:
     """Measure and return approximate number of cycles per millisecond for torch.cuda._sleep
 
-    Copied from: github.com/pytorch/pytorch/blob/master/test/test_cuda.py
+    Copied from: github.com/pytorch/pytorch/blob/main/test/test_cuda.py
```

Comment:
not sure about that actually, false positive ?

Reply:
Yes. They have not updated the branch name yet. I corrected it.